modules/lib.lua

```diff
@@ -419,6 +419,7 @@ else
     end
 
     ---Deletes the specified vehicle and returns whether it was successful.
+    ---@deprecated use exports.qbx_core:DeleteVehicle(vehicle) instead
```

Comment:
`qbx.deleteVehicle` is client-side. Should we still deprecate it in favor of a server-side export? Not all cars will necessarily be persistent.

Reply:
reverted the deprecation.